src/rviz_visual_tools.cpp

```diff
@@ -896,6 +902,10 @@ bool RvizVisualTools::publishMarkers(visualization_msgs::MarkerArray &markers)
   if (markers.markers.empty())
     return false;
 
+  if (frame_locking_enabled_)
+    for (auto &marker: markers.markers)
+      marker.frame_locked = true;
```

Comment:
I thought there were two separate markers type but I was wrong.
This was redundant.

I removed the loop and updated my PR.